.changeset/tasty-rockets-mix.md

```diff
@@ -0,0 +1,7 @@
+---
+"@khanacademy/wonder-blocks-dropdown": major
+---
+
+1. Updates dropdown openers for SingleSelect and MultiSelect to use `role="combobox"` instead of `button`.
+2. SingleSelect and MultiSelect now also require a paired `<label>` element or `aria-label` attribute for accessibility. They no longer fall back to text content for labeling, as those contents are now used as combobox values.
```

Comment:
small nit: Might be worth rephrasing the "required" part a bit since we aren't changing the API to enforce this (aria-label is still optional and we don't have anything in the component checking if there is an associated label element). Maybe: 
```suggestion
2. SingleSelect and MultiSelect should have a paired `<label>` element or `aria-label` attribute for accessibility. They no longer fall back to text content for labeling, as those contents are now used as combobox values.
```